magento/pwa-studio
# [Docs] Add adobe logo

## Description

This PR adds Adobe logo to the header. 

It also fixes a bug with columns on the home page

## Related Issue





Closes #ISSUE_NUMBER.

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to docs home page
2. See Adobe logo in top left
3. See tiles in 3 columns and not 1.

## Screenshots / Screen Captures (if appropriate)

![Screen Shot 2020-10-22 at 3 18 28 PM](https://user-images.githubusercontent.com/752610/96925494-ddd11e80-1479-11eb-9f75-c0ad5c7344c1.png)


## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
